documentation/sphinx/source/perpetual-storage-wiggle.rst

```diff
@@ -0,0 +1,56 @@
+.. _perpetual-storage-wiggle:
+
+############################
+Perpetual Storage Wiggle
+############################
+
+.. include:: guide-common.rst.inc
+
+This document covers the concept and usage of perpetual storage wiggle.
+
+.. _perpetual-storage-wiggle-introduction:
+
+Summary
+============
+Perpetual storage wiggle is a feature that lets the data distributor is forced to constantly build new storage teams when the cluster is healthy. On a high-level note, the process is like this:
+
+Order storage servers by process id. For each storage server n:
+
+a. Exclude storage server n.
+
+b. Wait until all data has been moved off the storage server.
+
+c. Include storage n
```

Comment:
I assume storage will get a different process id so that next candidate is different. If so, may be worth explicitly mentioning it.